osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -488,7 +468,7 @@ private void updateSpecifics()
                 UserStyleSection.Hide();
         }
 
-        protected virtual APIMod[] GetGameplayMods() => UserMods.Value.Select(m => new APIMod(m)).Concat(SelectedItem.Value!.RequiredMods).ToArray();
```

Comment:
This is overridden by MultiplayerMatchSubScreen (its only inheritor). Please overlook for the time being because it's going away.